sql/startup_Date_time.sql

```diff
@@ -0,0 +1,10 @@
+-- last_startup_time.sql
+-- Laurent L - zoltix
+-- 2020-01-31
+-- get Last Starstup Date time
+
+select 
+    to_char(startup_time, 'HH24:MI DD-MON-YY') "Startup time"
```

Comment:
Hard coded date formats should be in ISO 8601 format

ie.  'YYYY-MM-DD HH24:MI:SS' works for dates

'YYYY-MM-DD HH24:MI:SSXFF' is good for timestamps